tutorials/preprocessing/20_rejecting_bad_data.py

```diff
@@ -49,7 +49,7 @@
 # existing annotation labels can be selected for use.
 
 fig = raw.plot()
-fig.canvas.key_press_event('a')
+fig._fake_keypress('a')
```

Comment:
Hmm... we shouldn't use private methods in tutorials. We can make this public I think